ansible/vars/dev/dev_public.yml

```diff
@@ -5,6 +5,7 @@ COMMTRACK_SITE_HOST: '{{ groups.proxy.0 }}'
 NO_WWW_SITE_HOST: '{{ groups.proxy.0 }}'
 J2ME_SITE_HOST: 'j2me.{{ groups.proxy.0 }}'
 internal_domain_name: 'localdomain'
+ubuntu_version: "14.04"
```

Comment:
I think you can use `ansible_distribution_version` for this

Reply:
Thanks I didn't know about that variable. much cleaner now